README.md

```diff
@@ -1,2 +1,72 @@
 # strike-off-objections-api
+
+### Overview
 API for handling objections to a company being struck off the register
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+
+### Getting started
+
+To checkout and build the service:
+1. Clone [Docker CHS Development](https://github.com/companieshouse/docker-chs-development) and follow the steps in the README.
+2. Run ./bin/chs-dev modules enable strike-off-objections
+3. Run ./bin/chs-dev development enable strike-off-objections-api (this will allow you to make changes).
+4. Run docker using "tilt up" in the docker-chs-development directory.
+5. Use spacebar in the command line to open tilt window - wait for strike-off-objections-web to become green.
+6. Open your browser and go to page http://chs.local/strike-off-objections
+7. Work though the pages using example company THE BEAR RETAIL 05916434
+
+These instructions are for a local docker environment.
+
+
+### Endpoints
+
+Method    | Path                                                                         | Description
+:---------|:-----------------------------------------------------------------------------|:-----------
+**GET**   | `/eligibility`       | Returns information describing the strike-off eligibility of the company.
```

Comment:
These paths should be prefixed with ${API_URL}/company/{COMPANY_NUMBER}/strike-off-objections/

Reply:
That would make the paths very long and more difficult to tell each one apart as it is the same for all paths then it would be better to mention this above.